dss/config.py

```diff
@@ -4,23 +4,37 @@
 
 from .blobstore import BlobStore
 from .blobstore.s3 import S3BlobStore
+from .blobstore.gcs import GCSBlobStore
 from .hcablobstore import HCABlobStore
 from .hcablobstore.s3 import S3HCABlobStore
+from .hcablobstore.gcs import GCSHCABlobStore
 
 
 class Config(object):
     _S3_BUCKET = None  # type: str
+    _GS_BUCKET = None  # type: str
 
     @staticmethod
     def get_cloud_specific_handles(replica: str) -> typing.Tuple[
             BlobStore, HCABlobStore, str]:
+
+        handle: BlobStore
```

Comment:
handle.

type hinting is currently broken in python if the same variable is declared in two different paths of an if-statement.